projects/netgrif-components-core/src/lib/panel/task-panel/abstract-task-panel.component.ts

```diff
@@ -320,7 +326,7 @@ export abstract class AbstractTaskPanelComponent extends AbstractPanelWithImmedi
     }
 
     public canCollapse(): boolean {
-        return this._permissionService.canCollapse(this.taskPanelData.task);
+        return this._taskForceOpen ? false : this._permissionService.canCollapse(this.taskPanelData.task);
```

Comment:
How does this force open work? If it is set to true, it is not possible to close the task?